### PR TITLE
Fix: Beautify files site wide

### PR DESCRIPTION
**Pull request by varun13169**

### CODE REVIEW

File: library/card/card.css [hunk 1]
```diff
@@ -23,13 +23,13 @@
     grid-area: cardActions;
 }
 
-.dui-card__info {
-}
+.dui-card__info {}
```
Comment (zoef-yt) [inline library/card/card.css:26]: can be removed if not in use.

File: library/card/card.css [hunk 2]
```diff
@@ -46,8 +46,7 @@
 
 }
 
-.dui-card__button {
-}
+.dui-card__button {}
```
Comment (zoef-yt) [inline library/card/card.css:49]: can be removed it not in use.

**Review by Shubhamkr2610 — APPROVED** [review]
Looks good to me . You can merge

File: reset-css.css [hunk 3]
```diff
@@ -2,5 +2,4 @@
     box-sizing: border-box;
     padding: 0px;
     margin: 0px;
```
Comment (zoef-yt) [inline reset-css.css:4]: px units can be removed

File: styles.css [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
 
-:root{
+:root {
     --grey-secondary: #757575;
 
 }
-body{
+
+body {
     background: #dedede;
```
Comment (zoef-yt) [inline styles.css:10]: use var

File: styles.css [hunk 5]
```diff
@@ -84,14 +89,11 @@ body{
...
-}
+.header__btn--download {}
 
-.header__btn--print {
-}
+.header__btn--print {}
 
-.header__btn--save {
-}
+.header__btn--save {}
```
Comment (zoef-yt) [inline styles.css:96]: can be removed if not in use

**Review by DivyanshSareen — APPROVED** [review]
Good to merge

File: styles.css [hunk 6]
```diff
@@ -116,13 +119,12 @@ body{
...
 }
+
 .drawer__heading {
     margin: 0rem 0rem 1.6rem 0rem;
 }
 
-.drawer__title {
-
-}
+.drawer__title {}
```
Comment (zoef-yt) [inline styles.css:127]: can be removed if not in use

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: library/card/card.css (hunk 1 above)
Comment (varun13169) [reply library/card/card.css:26]: No, that's a placeholder for future use.

File: library/card/card.css (hunk 2 above)
Comment (varun13169) [reply library/card/card.css:49]: No, that's a placeholder for future use.

File: reset-css.css (hunk 3 above)
Comment (varun13169) [reply reset-css.css:4]: Explicit is better.

File: styles.css (hunk 6 above)
Comment (varun13169) [reply styles.css:127]: No, that's a placeholder for future use.

File: styles.css (hunk 5 above)
Comment (varun13169) [reply styles.css:96]: No, that's a placeholder for future use.

**Review by Shubhamkr2610 — APPROVED** [review]
looks good to me

File: styles.css (hunk 4 above)
Comment (varun13169) [reply styles.css:10]: Specific to documentation site.
Only being used once, hence didn't use variables.

**Pull request merged**